components/director/pkg/graphql/formation_constraint_validation.go

```diff
@@ -155,11 +155,11 @@ func emptySendNotificationOperationDetails() *formationconstraint.SendNotificati
 			CreatedAt: &time.Time{},
 		},
 		TemplateInput: nil,
-		FormationAssignment: &model.FormationAssignment{
-			Value: json.RawMessage("\"\""),
+		FormationAssignment: &webhook.FormationAssignment{
+			Value: "\"\"",
```

Comment:
Do we need this to be `"\"\""` when we have the webhook.FA and its .Value field of string?
P.S. Valid for the other occurrence as well